### PR TITLE
getconf: add page

### PR DESCRIPTION
**Pull request by oytv**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/getconf.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
+>
```
Comment (blueskyson) [inline pages/linux/getconf.md:4]: Please add more information link.

File: pages/linux/getconf.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
+> 
+
+- List every configuration values available
```
Comment (blueskyson) [inline pages/linux/getconf.md:6]:
```suggestion
- List every configuration values available:
```

File: pages/linux/getconf.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`getconf -a`
+
+- List the configuration values for a specific directory
+
+`getconf -a {{directory}}`
```
Comment (blueskyson) [inline pages/linux/getconf.md:12]:
```suggestion
- List the configuration values for a specific directory:

`getconf -a {{path/to/directory}}` 
```

File: pages/linux/getconf.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- List every configuration values available
+
+`getconf -a`
+
+- List the configuration values for a specific directory
+
+`getconf -a {{directory}}` 
+
+- Check if your linux system is a 32- or 64-bit system
```
Comment (blueskyson) [inline pages/linux/getconf.md:14]:
```suggestion
- Check if your linux system is a 32- or 64-bit system:
```

File: pages/linux/getconf.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`getconf LONG_BIT`
+
+- Check how many processes a user can run at once
```
Comment (blueskyson) [inline pages/linux/getconf.md:18]:
```suggestion
- Check how many processes a user can run at once:
```

File: pages/linux/getconf.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`getconf CHILD_MAX`
+
+- List every configuration value and then find patterns with the grep command. i.e every value with MAX in it.
+
+`getconf -a | grep MAX`
```
Comment (blueskyson) [inline pages/linux/getconf.md:24]: I think this example is not necessary.
```suggestion
```

File: pages/linux/getconf.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# getconf
+
+> getconf - get configuration values from your Linux system.
```
Comment (EmilySeville7cfg) [inline pages/linux/getconf.md:3]:
```suggestion
> getconf - get configuration values from your Linux system.
> More information: https://manned.org/getconf.1.
```

- This change is [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
- More information should be provided to allow user explore command in details when needed.

File: pages/linux/getconf.md (hunk 2 above)
Comment (EmilySeville7cfg) [inline pages/linux/getconf.md:6]:
```suggestion
- List [a]ll configuration values available:
```

- This change is optional - no corresponding style guide requirement exists.
- We often provide mnemonics for short options to permit users easy memorize them.

File: pages/linux/getconf.md (hunk 3 above)
Comment (EmilySeville7cfg) [reply pages/linux/getconf.md:12]: Some additions to a suggestion above:

- This change is [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
- Path format should be standardized across all pages.

File: pages/linux/getconf.md (hunk 5 above)
Comment (EmilySeville7cfg) [inline pages/linux/getconf.md:18]:
```suggestion
- Check how many processes the current user can run at once:
```

- This change is partially [mandatory](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#layout).
- Mandatory: trailing colons should be always provided.
- Optional: it's better to specify whether we are talking about the current user or not.

File: pages/linux/getconf.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+- Check if your linux system is a 32- or 64-bit system
+
+`getconf LONG_BIT`
+
+- Check how many processes a user can run at once
+
+`getconf CHILD_MAX`
+
+- List every configuration value and then find patterns with the grep command. i.e every value with MAX in it.
```
Comment (EmilySeville7cfg) [inline pages/linux/getconf.md:22]:
```suggestion
- List every configuration value and then find patterns with the grep command. i.e every value with MAX in it:
```

Almost the same issue as above.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

File: pages/linux/getconf.md (hunk 7 above)
Comment (sbrl) [reply pages/linux/getconf.md]: Teminology such as *mandatory* sounds intimidating. Perhaps would could rephrase that?

Comment (EmilySeville7cfg) [reply pages/linux/getconf.md]: I don't know. But I always divide fixes into two categories: mandatory (required by style guide or specification) and option (all others).

**Pull request merged**